tutorials/networking/high_level_multiplayer.rst

```diff
@@ -103,388 +101,280 @@ for full IPv6 support.
 Initializing the network
 ------------------------
 
-The object that controls networking in Godot is the same one that controls everything tree-related: :ref:`SceneTree <class_SceneTree>`.
-
-To initialize high-level networking, the SceneTree must be provided a NetworkedMultiplayerPeer object.
+High level networking in Godot is managed by the :ref:`SceneTree <class_SceneTree>`.
 
-To create that object, it first has to be initialized as a server or client.
+Every node has a ``multiplayer`` property, which is a reference to a default ``MultiplayerAPI`` instance.
 
-Initializing as a server, listening on the given port, with a given maximum number of peers:
+It is possible to create a new ``MultiplayerAPI`` object and assign it to a ``NodePath`` in the the scene tree,
+which will override ``multiplayer`` for the node at that path and all of it's children.
+This allows sibling nodes to be configured with different peers, which makes it possible to run a server
+and a client simultaneously in one instance of Godot.
 
 ::
 
-    var peer = NetworkedMultiplayerENet.new()
-    peer.create_server(SERVER_PORT, MAX_PLAYERS)
-    get_tree().network_peer = peer
+    # By default, these expressions are equivalent.
+    multiplayer
+    get_tree().get_multiplayer()
```

Comment:
Again, only if none has been configured, this is going to confuse people

Reply:
That's what I put the "by default" for, but I can see it not being clear enough. I'm open to suggestions for better language since I'm not too sure.

Reply:
Why is this relevant? It's only true for a very specific case, so it's unhelpful information

Reply:
I wanted to specifically mention that `Node.multiplayer` is a reference to a "global" object, and not one unique per node. I've seen people ask why you cannot set this property even though you can get it. When learning the API I was myself confused about why one can just get multiplayer from every node and receive the same object.

Reply:
Then the correct code is `get_tree().get_multiplayer(get_path())`

Reply:
I believe this should stay mentioned, otherwise the example I placed below is confusing. Why is `multiplayer` getting set in a singleton, but works for every node? I understand it's documented in the class reference, but I think it's worth for new users to see.

Reply:
Sure, but the current wording is far too confusing and will cause people to assume this is the correct way of getting the `MultiplayerAPI` of a `Node`, which is bad

Reply:
I did not understand which is the correct way. I assume it's through the `multiplayer` property, because why would it exist otherwise.

It makes sense to me that the preferred way should be mentioned in the docs.

Reply:
The correct way is `multiplayer`, it ensures you are in the tree

However calling `get_tree().get_multiplayer()` returns the default one, not the one associated with any one node

Reply:
Edited these to be more clear.

Reply:
This is still going to confuse people, why not use the proper equal expression as an example?

Reply:
I'm not willing to die on this hill... for now I want to see if a networking maintainer approves of the content generally before I put more effort into polishing it.

Reply:
I'm sorry if this was unpleasant, I didn't mean for it to become an argument, my apologies

Reply:
I find  `get_tree().get_multiplayer()` to be overly verbose. `get_multiplayer()` works on its own and makes more sense in my opinion. This is important for a tutorial.

Reply:
Those are not equivalent, as is the entire point of this discussion here

Reply:
> Those are not equivalent, as is the entire point of this discussion here

I'm not saying that they are. I'm saying I'm not fond of using `get_tree().get_multiplayer()` for the reasons explained. If people really believe it's a must, I believe further explanation is necessary. Because otherwise, people will have the same doubts that we are having in this discussion.
Reasons to use `get_tree().get_multiplayer()` should be clear if used in my opinion.

Reply:
I'm sorry your original comment made it seem that way based on the suggestion of replacing one with the other

Reply:
I see, no worries. It was confusing, the way I put it. My fault as well.
Always please feel free to argue my posts. I'm new here, so I appreciate a lot the feedback. Hopefully I may contribute to the discussion.